capa/rules.py

```diff
@@ -577,7 +574,10 @@ def build_statements(d, scope: str):
         Feature = parse_feature(key)
         value, description = parse_description(d[key], key, d.get("description"))
         try:
-            feature = Feature(value, description=description)
+            if key == "property/read" or key == "property/write":
+                feature = Feature(value, key, description=description)
+            else:
+                feature = Feature(value, description=description)
```

Comment:
Ugh, as you said this isn't great... 
As a workaround, let's move this to an `elif` block above and return the feature there.

Reply:
Alright